doc/articles/features/uno-material.md

```diff
@@ -77,7 +77,7 @@ Prior to the `1.0` release, the initialization of Material resources was require
 Control|Resource Key
 -|-
 `AppBarButton`|MaterialAppBarButton
-`Button`|MaterialContainedButtonStyle<br>MaterialContainedSecondaryButtonStyle<br>MaterialFabStyle<br>MaterialOutlinedButtonStyle<br>MaterialOutlinedSecondaryButtonStyle<br>MaterialPaneBackArrowToggleButtonStyle<br>MaterialPaneToggleButtonStyle<br>MaterialPrimaryInvertedFabStyle<br>MaterialSecondaryFabStyle<br>MaterialSecondaryInvertedFabStyle<br>MaterialSmallFabStyle<br>MaterialTextButtonStyle<br>MaterialTextSecondaryButtonStyle
+`Button`|MaterialContainedButtonStyle<br>MaterialContainedIconButtonStyle<br>MaterialContainedSecondaryButtonStyle<br>MaterialContainedSecondaryIconButtonStyle<br>MaterialFabStyle<br>MaterialOutlinedButtonStyle<br>MaterialOutlinedIconButtonStyle<br>MaterialOutlinedSecondaryButtonStyle<br>MaterialOutlinedSecondaryIconButtonStyle<br>MaterialPaneBackArrowToggleButtonStyle<br>MaterialPaneToggleButtonStyle<br>MaterialPrimaryInvertedFabStyle<br>MaterialSecondaryFabStyle<br>MaterialSecondaryInvertedFabStyle<br>MaterialSmallFabStyle<br>MaterialTextButtonStyle<br>MaterialTextIconButtonStyle<br>MaterialTextSecondaryButtonStyle<br>MaterialTextSecondaryIconButtonStyle
```

Comment:
You need to explain somewhere how to actually use the icon button styles - maybe as a footnote at the bottom.

Reply:
Done! Hope this level of detail will suffice.
![image](https://user-images.githubusercontent.com/20365014/123142001-ff2e4400-d426-11eb-9caf-28e2a7e30761.png)

Reply:
This is good, but we also want a clear and complete explanation of how to use the Icon property and styles, with example code. 

Probably we don't want to squeeze it into the table though; maybe a section below, or it could go in the linked https://github.com/unoplatform/uno/blob/master/doc/articles/features/uno-material-controls-extra-setup.md document

Reply:
@davidjohnoliver I added a section below that contains a code example and tweaked my previous addition to the table's wording to tie them together.